SIT.Manager/Views/Tools/NetworkToolsView.axaml.cs

```diff
@@ -0,0 +1,31 @@
+using Avalonia.Controls;
+using Microsoft.Extensions.DependencyInjection;
+using SIT.Manager.Controls;
+using SIT.Manager.ViewModels.Tools;
+using System;
+using System.Threading;
+
+namespace SIT.Manager.Views.Tools;
+public partial class NetworkToolsView : ActivatableUserControl
+{
+    private readonly NetworkToolsViewModel _dc;
+    public NetworkToolsView()
+    {
+        InitializeComponent();
+        _dc = App.Current.Services.GetService<NetworkToolsViewModel>() ?? throw new Exception("meow");
+        this.DataContext = _dc;
```

Comment:
If you use GetRequiredService here instead then it won't return a nullable and throw and exception instead just like you're doing here.